src/transformStateWithClones.js

```diff
@@ -7,7 +7,40 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let copyState = { ...state };
```

Comment:
Let's change this to `const`.
We don't need to rewrite copyState everytime

Reply:
Check @33Orange comment, you didn't it

Reply:
> Check @33Orange comment, you didn't it

@Anna-Tkachenko But then I can`t add properties(( on lesson in the morning we used 'let'...

Reply:
now I finally can) interesting point for me: if I use const there + just Object.assign (without copyState = Object.assign....) for adding, everything works fine